Updates to billing FAQs (#3911)
dbt-labs/docs.getdbt.com: website/docs/faqs/Accounts/payment-accepted.md

id: payment-accepted
---

> Presently for Team plans, self-service dbt Cloud payments must be made via credit card and by default, they will be billed monthly based on the number of [active developer seats and usage](/docs/cloud/billing).

Reviewer: ```suggestion
Currently for Team plans, self-service dbt Cloud payments must be made with a credit card and by default, they will be billed monthly based on the number of [active developer seats and usage](/docs/cloud/billing).
```
